### DISCUSSION TITLE
Question Autostart

### DISCUSSION THREAD

**Discussion by PDotStrong** [rdudhagra/Sand-Table#47] [Q&A] [answered]
Hi I am again 🙈

last question so far! Is it possible to start with a specific tracklist, so that after restart or plug in the system automatically start tracks? Where I have to make it?

**Comment by rdudhagra** [comment]
You could try setting the initial value of `currentTracks` here to your desired playlist: https://github.com/rdudhagra/Sand-Table/blob/f8876251e55bee3187eb94d0f94dd297293fad03/Website/server/routes/track_manager.js#L10

**Reply by PDotStrong** [reply]
Ok, but I have to point to the playlist right? I cant write in the brackets for example autostart (the name of my playlist)?!? I have to say where the playlist ist stored or? And if yes I dont find it

**Reply by rdudhagra** [reply, marked as answer]
So the way I implemented playlists is that when you "play" a playlist, it just replaces the current queue of tracks with the content of the playlist. You could take the content of your autostart playlist and just set it to currentTracks—although, then you wouldn't be able to modify this list through the website.

**Reply by PDotStrong** [reply]
So I got it! The Playlist starts at beginning.

**Reply by PDotStrong** [reply]
And I still can make playlist etc
